fastlane/Fastfile

```diff
@@ -26,6 +26,7 @@ platform :android do
   lane :test do
     gradle(task: "test")
     gradle(task: "deviceCheck")
+    gradle(task: "app:connectedCheck")
```

Comment:
Let's add the leading colon: ":app:connectedCheck". Leaving out the colon on project/sub-project can be ambiguous